dispatcher/src/main/kotlin/pl/droidsonroids/testing/mockwebserver/YamlResourcesParser.kt

```diff
@@ -5,7 +5,7 @@ import org.yaml.snakeyaml.Yaml
 
 internal class YamlResourcesParser : ResourcesParser {
     private val escaper = JavaUnicodeEscaper.above(0xD800)
-    private val parser = Yaml()
+    private val parser get() = Yaml()
```

Comment:
This can can be inlined. It makes no sense to keep this field when we create a new parser each time.
Change `parser` in L14 to `Yaml()` and remove this field.

Reply:
Well, this acts like a builder, so if we need to create a new Yaml, we can just use this.

Anyway, this is a simple code and is not reused anywhere, so it make sense to inline it 👌